boards/arm/adafruit_feather_nrf52840/adafruit_feather_nrf52840_sense.dts

```diff
@@ -0,0 +1,167 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "feather_connector.dtsi"
+#include "adafruit_feather_nrf52840-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Adafruit Feather nRF52840 Sense";
+	compatible = "adafruit,feather-nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
```

Comment:
Should this be UART, or should this be USB CDC?

Reply:
I copied most of this from the Express variant. Since I haven't been able to flash the board (Sense) that I have, most of these options are opaque to me. I posted this PR to more articulately ask for help from people who *have* had success like @jefffhaynes. I'd also love to hear from @tsvehagen who originally added support for the Express board.

Reply:
> Since I haven't been able to flash the board (Sense) that I have

What is not working, surely the device should just load the uf2 file you have generated? Or do you mean that it does not run it?

Reply:
It does not run it: https://github.com/zephyrproject-rtos/zephyr/pull/63819#discussion_r1355416877

Reply:
I would assume that the bootloader is not compatible with bossac, would make more sense to support the uf2 runner. I've flashed their bootloader to a different board here and can confirm that the uf2 flash runner works though (in linux) the partition must be mounted for it to detect it

Reply:
Didn't see there was a runner for UF2. I've made that the default runner and removed BOSSA.

I still can't flash blinky however. `$ west flash` fails with:
<details>
<summary>(click for errors)</summary>
<pre>
-- west flash: rebuilding
ninja: no work to do.
-- west flash: using runner uf2
-- runners.uf2: Copying UF2 file to '/media/ben/FTHRSNSBOOT'
Traceback (most recent call last):
  File "/home/ben/mambaforge/envs/zephyr/bin/west", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/site-packages/west/app/main.py", line 10
85, in main
    app.run(argv or sys.argv[1:])
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/site-packages/west/app/main.py", line 24
4, in run
    self.run_command(argv, early_args)
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/site-packages/west/app/main.py", line 50
5, in run_command
    self.run_extension(args.command, argv)
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/site-packages/west/app/main.py", line 65
4, in run_extension
    self.cmd.run(args, unknown, self.topdir, manifest=self.manifest,
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/site-packages/west/commands.py", line 19
4, in run
    self.do_run(args, unknown)
  File "/home/ben/zephyrproject/zephyr/scripts/west_commands/flash.py", line 33, in do_run
    do_run_common(self, my_args, runner_args, domains=domains)
  File "/home/ben/zephyrproject/zephyr/scripts/west_commands/run_common.py", line 174, in do_run
_common
    do_run_common_image(command, user_args, user_runner_args, d.build_dir)
  File "/home/ben/zephyrproject/zephyr/scripts/west_commands/run_common.py", line 250, in do_run
_common_image
    runner.run(command_name)
  File "/home/ben/zephyrproject/zephyr/scripts/west_commands/runners/core.py", line 621, in run
    self.do_run(command, **kwargs)
  File "/home/ben/zephyrproject/zephyr/scripts/west_commands/runners/uf2.py", line 105, in do_ru
n
    self.copy_uf2_to_partition(part)
  File "/home/ben/zephyrproject/zephyr/scripts/west_commands/runners/uf2.py", line 88, in copy_u
f2_to_partition
    copy(self.cfg.uf2_file, part.mountpoint)
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/shutil.py", line 424, in copy
    copymode(src, dst, follow_symlinks=follow_symlinks)
  File "/home/ben/mambaforge/envs/zephyr/lib/python3.12/shutil.py", line 313, in copymode
    chmod_func(dst, stat.S_IMODE(st.st_mode))
FileNotFoundError: [Errno 2] No such file or directory: '/media/ben/FTHRSNSBOOT/zephyr.uf2'
</pre>
</details>

Reply:
You will need to use python 3.11, python 3.12 is not supported

Reply:
I didn't see that 3.12 is not supported. I've replaced my environment with on using 3.11, and I still get a very similar error. I've updated the above traceback just in case I missed something

Reply:
So that error looks like you do not have permission to copy files to that drive, can you manually copy the file and does it work?

Reply:
Yes, as I described [above](https://github.com/zephyrproject-rtos/zephyr/pull/63819#discussion_r1355416877) I can copy UF2 files produced with arduino IDE onto the drive. Also manually copying the UF2 file produced by west does not raise an error (`cp` exits with code 0).